manager.py

```diff
@@ -157,16 +160,24 @@ def get_log_entries(
             name (str): filename identifier (full filename is figured out by
                 adding project path and extension), if name is None, all
                 files in project's logs directory are considered
+            id (str): service identifier
             entries_count (int): number of entries to read (-1 reads them all)
             level (str): level to filter by
             component (str): component to filter by
 
         Returns:
              list of entries where items are in dict format
         """
-        if name:
-            if name not in self._service_list():
-                raise ValueError("{} service does not exist".format(name))
+        if name or id:
+            services = self._service_manager.services
+            if id is not None and name is None:
+                # figure out name from id
+                name = services.get(id, None)
+                if name is None:
+                    raise ValueError("Invalid service id specified")
+            else:
+                if name != 'main' and name not in services.values():
+                    raise ValueError("'{}' service does not exist".format(name))
```

Comment:
Does this imply that we can't have a service without a name? Looking up the name based on a given ID seems backwards. But the fact that the logger name is the service name makes this kind of funky. 

My suggestion would be to make the logger name for a service the service `label` (name only unless no name, then ID). Then lookup services by label rather than fixed to name.

Reply:
Good call!